x-pack/plugins/security/server/audit/audit_events.ts

```diff
@@ -223,6 +223,14 @@ export function savedObjectEvent({
     : `User has ${past} ${doc}`;
   const type = eventTypes[action];
 
+  if (
```

Comment:
Can we add a couple of tests to `x-pack/plugins/security/server/audit/audit_events.test.ts` to verify these scenarios? It's rather obvious what's happening, but for posterity it'd be good to have this "documented" in the form of tests.

Reply:
They're all in here: https://github.com/elastic/kibana/pull/82314/files#diff-bf2ffccabb5a9bbb0f1d7a47c9a389a647db0e2a7afa1a80b61dcb0c6a1676a2

Are there any cases you think are missing?

Reply:
@thomheymann I'm officially losing my mind! I could have sworn that VSCode was showing no changes to that file in this PR. What you have here is perfect, sorry for the noise.